lib/testers/rule-tester.js

```diff
@@ -378,8 +376,9 @@ class RuleTester {
                     throw new Error([`Schema for rule ${ruleName} is invalid:`, errors]);
                 }
             }
+            const ruleMap = linter.getRules();
 
-            validator.validate(config, "rule-tester", ruleMap.get.bind(ruleMap), new Environments());
+            validator.validate(config, "rule-tester/validate-ast", ruleMap.get.bind(ruleMap), new Environments());
```

Comment:
It's not a big deal, but I think `rule-tester/validate-ast` is the name of the rule within `RuleTester` that validates ASTs. `validator.validate` accepts the name of a config as an argument, not the name of a rule, so I think `rule-tester` is better here.